### ISSUE TITLE
Clarify "minimize" versus "disconnect/end" actions for user

### ISSUE THREAD

**Issue by rachelmcr** [tinkertinker/wp-calypso#34]
Currently, when a user has an active chat there is an "X" at the top of the chat to minimize it:

![screen shot 2016-08-31 at 1 05 25 pm](https://cloud.githubusercontent.com/assets/8658164/18128023/a650543c-6f7b-11e6-840a-8298237d43c3.png)

I'd suggest clarifying the UI so users understand that this only minimizes the chat. When I saw it, I assumed that clicking the "X" would end the chat and disconnect me from the operator, so I was hesitant to click it.

I would be more confident if it were a more familiar "minimize" icon. (If users also eventually have an option to end/disconnect the chat, that could be represented with the "X" icon alongside the minimize icon.)

**Comment by Lochlaer** [comment]
If it's a fairly long chat, it would be better if the X/- was located in the bottom right of the chat box rather than at the top. It wasn't super obvious to me while I was chatting with @amylaneio where to do just that. 

I also want to 👍 the option to offer the users (not just HEs) a way to end the chat themselves. When I've used live chat with other services, I usually just want to ask my question, get an answer, and then close the chat immediately so that the window or pop-up is gone and my desktop is clean.

**Comment by andreabadgley** [comment]
cc @jasmussen

**Comment by jasmussen** [comment]
I don't know whether to comment here, or on #46, because this (excellent) ticket highlights some assumptions we've made on how the tech would work, which may turn out to not be practical. Or they just have to be in the future.

Here are the reasons why the design looks like it currently does. Basically, you're having a never-ending conversation with _WordPress.com_, and just like there isn't a concept of "ending a chat" in iMessage, we haven't so far thought of us needing a way to "end a chat" in Happychat. 

You'd chat about your business...

![screen shot 2016-09-29 at 10 44 32](https://cloud.githubusercontent.com/assets/1204802/18947073/deb68aa4-8631-11e6-9b51-924fd87e9df1.png)

... and when you're done you close the chat window, just like you'd close iMessage, or click the home button on an iPhone. 

When you need to ask new questions, or if an HE has a followup message to send the user, a little notification would show up where the (in-the-future permanent) Support Chat button sits:

![screen shot 2016-09-29 at 10 44 00](https://cloud.githubusercontent.com/assets/1204802/18947132/1b1df84c-8632-11e6-8c58-6232345aef62.png)

I recite this not to be bullish about that's what we have to do, rather to inform how we got here. It also leads to the fact that "fixing this ticket" is not as easy as replacing the X with a minimize icon. The minimize icon is a lowered rectangle that implies the _panel_ will be minimized into a small blue sliver at the bottom, just like how Olark works. Only that's not what's happening here. 

To summarize, in a way this ticket suggests we be "more like Olark". Which is not a bad thing per se — and it may be the pragmatic thing to do to get this out the door. But it is a sideshow to our original vision. Incidentally that suggests to me we should do this:
1. Revert the sidebar footer support chat link, reinstate the blue floating tab in the bottom
2. Change the icon to a minimize bar, and have it collapse the panel
3. Have some way for the user to end the chat.

1 and 2 are not as much work as they sound, but 3 is. Question is whether 3 is necessary for beta.

**Comment by rachelmcr** [comment]
Thanks for the background, @jasmussen! I don't think users really _need_ a way to end a chat from their end, especially given our goal to keep the user's chat history available whenever they open chat. As long as they can close the window and move on when they're done with the chat (which they can do now) it seems ok to me.

I also don't think a minimize icon/floating tab is absolutely necessary. I really do like the original vision and the design with the Support Chat button in the sidebar.

The main issue I felt while I was using the chat (and trying to temporarily dismiss it to get more screen real estate) was fear that clicking the "X" would close/end the session prematurely. It was the associations I have with "X" (i.e. close an app, dismiss a notification) that led to my fear. I might not be representative so I wouldn't want to suggest dramatic changes to the chat behavior based on just my reaction, but it's perhaps something we should keep an eye out for during user testing.

**Comment by jasmussen** [comment]
Thanks for the followup Rachel. 

As an alternative to reverting back to the panel mode, is to try and stick to what we have, but change the X button to a minimize icon — perhaps a different one, maybe a downward chevron? But then we'd need the notification dot from #57.

**Comment by andreabadgley** [comment]
Rachel definitely was not the only one who felt the X was very final. Several people in testing asked "What does the X do?" They asked because they didn't want to click it, fearing it would close the chat.

I vote for a downward chevron and the notification dot. Beau indicated in the call yesterday that the dot might not be too much work for beta. :fingerscrossed:

**Comment by kriskarkoski** [comment]
For reference, here's what Olark looks like now:

<img width="320" alt="screen shot 2016-09-29 at 12 10 03 pm" src="https://cloud.githubusercontent.com/assets/1900444/18962625/9d22c4aa-863f-11e6-9ca6-58eed9ff1c6c.png">

Clicking that option collapses chat into a small chat icon in the bottom right corner:

<img width="67" alt="screen shot 2016-09-29 at 12 07 39 pm" src="https://cloud.githubusercontent.com/assets/1900444/18962641/b196012c-863f-11e6-9384-4dad0f91c7f0.png">

**Comment by jasmussen** [comment]
This PR here changes the X to a downwards chevron. It also adds a "minimize" animation, hopefully directing the user attention to the button they have to press to open the chat again. I feel like the combination of those two, once merged in, should close this issue. 

Edit: forgot to specify which PR I was referencing. It's this: https://github.com/Automattic/wp-calypso/pull/8706

**Comment by jasmussen** [comment]
This is now merged in!
